### PR TITLE
Data sync wt local setup master

### PR DESCRIPTION
**Pull request by mmusil**
Updated the walkthrough for the latest version of datasync-starter.

CC @wtrocki

### CODE REVIEW

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 1]
```diff
@@ -113,13 +119,12 @@ and available plugins that will be used for source code generation
 `yarn graphback generate`
 . Review `./server/src/schema/schema.qraphql`.
 This file contains your model along with access methods that were enabled in configuration.
-. Review generated resolver files in `./server/src/resolvers/resolvers.ts`
 This file contains methods used to fetch data. Each individual method will use
```
Comment (wtrocki) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: Remove this sentence.

**Review by wtrocki — APPROVED** [review]
Amazing work! Thank you so much!
Going to release new walkthrough branch after this is meged.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc (hunk 1 above)
Comment (mmusil) [reply walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: removed in 95370f7376c73c14c90b1975352d1e38b20507e2

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 2]
```diff
@@ -74,25 +74,31 @@ and generate underlying backend that will work out of the box with the RHMI serv
 DataSync model offers capability to generate underlying GraphQL API and database infrastructure.
 Starter template by default will use MongoDB database as datasource for storing all your data.
 
-. In your project go to `./model/task.graphql` file.
+. In your project go to `./server/task.graphql` file.
 This file contains GraphQL compatible schema
-. Please add `@datasync` annotation to the model right below `@model` annotation
-Those annotations can be used to control various behaviour of datasync.
+. Please add `@datasync` annotation to the model right below `@model` annotation.
+This annotation will enable data synchronization for this particular object. You can read more about data synchronization here https://graphback.dev/docs/datasync/soft-delete
```
Comment (craicoverflow) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc:80]:
```suggestion
This annotation will enable data synchronization for this particular object. You can read more about data synchronization here: https://graphback.dev/docs/datasync/soft-delete
```

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 3]
```diff
@@ -74,25 +74,31 @@ and generate underlying backend that will work out of the box with the RHMI serv
...
 . Your task model should look as follows
 ----
-"""
-@model
-@datasync
-"""
 type Task {
-  id: ID!
+  """@id"""
+  id: ObjectID!
```
Comment (craicoverflow) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc:88]: > If you are using MongoDB you should use `_id: ObjectID`, as `_id` in MongoDB documents is reserved for the primary key.

I checked the code for 0.14.x and it does not throw an error by design (it does in 0.15.x)..but maybe we should follow what the docs says and use `_id: ObjectID`?

https://graphback.dev/docs/0.14.x/model/datamodel#mongodb

**Review by craicoverflow — COMMENTED** [review]
I left one comment regarding the use of `id: ObjectID` - though it is not an issue for 0.14.0 as it is valid, the docs do recommend using `_id: ObjectID`.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc (hunk 3 above)
Comment (craicoverflow) [reply walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc:88]: Actually, the docs is also a little bit wrong (you still need to include the `@id` annotation but does does not specify this.

**Review by craicoverflow — APPROVED** [review]

**Comment by wtrocki** [comment]
Good to merge.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc (hunk 3 above)
Comment (wtrocki) [reply walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc:88]: This is from datasync starter that wasn't updated to latest 0.15.0 yet.

**Pull request merged**